config/management/operational/src/account_resource.rs

```diff
@@ -121,8 +123,14 @@ impl RotateOperatorKey {
         let rotate_key_txn = Transaction::UserTransaction(rotate_key_txn);
 
         // Submit the transaction
-        let txn_ctx =
+        let mut transaction_context =
             client.submit_transaction(rotate_key_txn.as_signed_user_txn().unwrap().clone())?;
-        Ok((txn_ctx, new_storage_key))
+
+        // Perform auto validation if required
+        transaction_context = self
+            .auto_validate
+            .execute(config.json_server, transaction_context)?;
+
+        Ok((transaction_context, new_storage_key))
```

Comment:
Btw, the build on top of the previous commits thing is kinda weird, but I do like you separated it out by which commits

Reply:
Thanks @gregnazario! Yeah, I was kind of hoping that would be reviewed first.. so we'd ultimately not need to worry about those commits :P